### PR TITLE
Hide input option, customize height plus fix for styled-components

### PR DESCRIPTION
**Pull request by gamedevsam**
I wasn't able to use `styled-components` in my app because they overwrote the ones in `react-simple-chatbot`. I fixed that issue by adding `'styled-components'` to the `except` option in webpack.prod.js.

### CODE REVIEW

**Comment by LucasBassetti** [comment]
Hi Sam,

I appreciate if you can write some tests for this `hideInput` proposal.

**Comment by gamedevsam** [comment]
I've never done test driven development before - it'll be a fun challenge!

It seems the test should go in [ChatBot.spec.js](https://github.com/LucasBassetti/react-simple-chatbot/blob/master/tests/lib/ChatBot.spec.js). 

It might come in sometime next week since I'm very busy this weekend.

Thanks for your input!

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```
Comment (gamedevsam) [inline .gitattributes]: I had to add this to fix eslint errors due to Windows changing line endings. More info here: https://stackoverflow.com/questions/9976986/force-lf-eol-in-git-repo-and-working-copy

**Comment by gamedevsam** [comment]
Test was added, thanks for the suggestion!

**Comment by LucasBassetti** [comment]
Great. I will check this ASAP. Thanks!

**Pull request merged**